corgi/core/models.py

```diff
@@ -515,13 +519,16 @@ def manifest(self) -> str:
         """Return an SPDX-style manifest in JSON format."""
         return ProductManifestFile(self).render_content()
 
-    def get_latest_components(self):
-        """Return root components from latest builds."""
+    def get_latest_components(self, start_dt=now()):
```

Comment:
This `now()` call gets evaluated once on start time and then it'll stay the same until you restart. I don't think that's what we want. Mutable arguments are usually passed as:

```python
def get_latest_components(self, start_dt=None):
    if start_dt is None:
        start_dt = now()
```

Reply:
ug ... thx for that ! (mutters: too many dynamic languages ...)

Reply:
removed start_dt